src/app/private/[uid]/chapter-leader/seniors/page.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { prisma } from "@server/db/client";
+import { faUsers } from "@fortawesome/free-solid-svg-icons";
+import { HeaderContainer } from "@components/container";
+import { SeniorView } from "@components/SeniorView";
+
+const UserSeniorsPage = async ({ params }: { params: { uid: string } }) => {
+  const userUid = params.uid;
+  const user = await prisma.user.findUnique({
+    where: {
+      id: userUid,
+    },
+  });
+  if (!user) {
+    return <div>User not found</div>;
+  }
+  // TODO: Maybe check if the user is actually a chapter leader
+  if (user.ChapterID === null) return <div>Does not belong to a chapter</div>;
```

Comment:
I think we can omit this check. We will eventually need a middleware to make sure that any access to `admin/`, `chapter-leader/`, and `user/` is protected.